pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -345,6 +345,10 @@ func AddAlertmanagerConfigMigrations(mg *migrator.Migrator) {
 	mg.AddMigration("add configuration_hash column to alert_configuration", migrator.NewAddColumnMigration(alertConfiguration, &migrator.Column{
 		Name: "configuration_hash", Type: migrator.DB_Varchar, Nullable: false, Default: "'not-yet-calculated'", Length: 32,
 	}))
+
+	mg.AddMigration("add successfully_applied column to alert_configuration", migrator.NewAddColumnMigration(alertConfiguration, &migrator.Column{
```

Comment:
Can we instead put this migration way up top, at the end of `AddTablesMigrations`?

If we put it here, it won't run in the right order. This will actually end up executing before other older migrations.

For the reason why, see: https://github.com/grafana/grafana/issues/60196